packages/staticcheck/package.yaml

```
@@ -10,7 +10,7 @@ categories:
   - Linter
 
 source:
-  id: pkg:golang/honnef.co/go/tools/cmd/staticcheck@2023.1.3
+  id: pkg:golang/honnef.co/go/tools/cmd/staticcheck@v0.4.6
```

Actually, we might as well change the PURL so that it targets the canonical package name (this might be why Renovate didn't manage to update):

```suggestion
  id: pkg:golang/honnef.co/go/tools@v0.4.6#cmd/staticcheck
```